docs/index.rst

```diff
@@ -4,6 +4,10 @@ AirGun
 AirGun is a Python library which is build over `Widgetastic`_ and `navmazing`_
 to make Satellite 6 UI testing easier.
 
+AirGun is written in Python 3 and do not plan to support Python 2. AirGun
+supports Satellite 6.4 and above and Foreman 1.17 and above (with vertical
+navigation).
```

Comment:
Btw it's currently possible to use py2 - backward compatibility is preserved atm, but yeah, it can be dropped any time, so i'm not sure if that's worth specifying, just fyi.

Reply:
I am also against specifying any static information like python or application versions. Usually, documentation is not updated in next years and such details become outdated

Reply:
IMO this Python version have same validity as info that only chromedriver is supported - it helps whoever is working on the project to set expectations. Same for Sat/Foreman versions: it only says minimal version which were set by management so we should let everybody know not to spend time trying to support e.g. Sat 6.3. But I have absolutely no problem with removing it from docs. Let me know.